features/khost/exec.config

```
@@ -3,11 +3,20 @@ set -Eeuo pipefail
 
 K8S_VERSION=1.25.3
 CRI_VERSION=1.25.0
+CHECKSUM_KUBEADM="01b59ce429263c62b85d2db18f0ccdef076b866962ed63971ff2bd2864deea7b"
```

bash specific question: do the quotes make any difference here? coming from a java background I always have trouble knowing what quotes mean in posix-shells, but I think leaving them out here would not make a difference

---

Quotes if it can be empty or contain spaces, otherwise quotes are not required at value assignment. 
Let me remove them for consistency/style
```suggestion
CHECKSUM_KUBEADM=01b59ce429263c62b85d2db18f0ccdef076b866962ed63971ff2bd2864deea7b
```